### ISSUE TITLE
3.2

### ISSUE THREAD

**Issue by nex86** [cbepx-me/Anbernic-H700-RG-xx-StockOS-Modification#89]
is 3.2 only for RG34XX ? or how do I understand this?

**Comment by Julioevm** [comment]
Yeah, I see rg35h mentioned in the changes but the download just mentions rg34xx.
Is it just compatible with both model series or this one is just for rg34xx?
